### DISCUSSION TITLE
Mi pokedex

### DISCUSSION THREAD

**Discussion by JosueDanieLB** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#501] [Show and tell]
### Cualquier recomendación es bien recibida, saludos!

[Página](https://josuedanielb.github.io/JavaScriptJosueFrontEnd-Mision/pokedexJosue.html)

[Repositorio](https://github.com/JosueDanieLB/JavaScriptJosueFrontEnd-Mision)

**Comment by OscarPedroza** [comment]
Muy buen trabajo compañero! excelente presentación.
